helmfiles/subhelmfiles/jenkins.helmfile.yaml

```diff
@@ -45,11 +49,18 @@ releases:
           jenkinsAdminEmail: {{$jenkinsAdminEmail | quote}}
           installPlugins:
             - kubernetes:1.25.7
+            - ace-editor:1.1
```

Comment:
I'm not familiar with some of these plugins. Just making sure none of them were added for testing and just got missed getting cleaned up.

Reply:
As mentioned in meeting, some of the plugins were added to support the Gitlab plugin and Gitlab OAuth plugin. They were consistently not being found during installation, so calling them out in the installPlugins element resolved the issue.